Improve ping level documentation (#550)
mozilla/firefox-data-docs: src/datasets/pings.md

# Raw Ping Data

> > **⚠** This article discusses pings sent by Firefox's legacy telemetry system.

Reviewer: ```suggestion
> **⚠** This article discusses pings sent by Firefox's legacy v4 telemetry system.
```

Firefox has had a few legacy telemetry systems; this is the one we still use. :D

<!-- table generated via `scripts/new_ping_metadata_table.py > src/cookbooks/new_ping_metadata_table.md` -->

{{#include ../cookbooks/new_ping_metadata_table.md}}

## Analysis

> The large majority of analyses can be completed using only the
> [main ping][main_ping].
> This ping includes histograms, scalars, and other performance and diagnostic data.

Reviewer: ```suggestion
The [main ping][main_ping] includes histograms, scalars, and other performance and diagnostic data.
Since Firefox 62, it [no longer contains event data](https://bugzilla.mozilla.org/show_bug.cgi?id=1460595); events are now sent in a separate `event` ping.
```

> Few analyses actually rely directly on any raw ping data.
> Instead, we provide [derived datasets](./derived.md) which are processed versions of these data,
> made to be:

Reviewer: Definitely untrue as written :)

```suggestion
[Derived datasets](./derived.md) are processed from ping tables. They are intended to be:
```

- Easier and faster to query
- Organized to make the data easier to analyze

> - Cleaned of erroneous or misleading data

Reviewer: Not to keep hammering this but I think we should avoid implying that e.g. clients_daily is generally "safe". I buy 1 and 2, though.

Author: Maybe we should just take out 3?

```suggestion
```

Reviewer: sgtm!

Ping data lives in BigQuery and is accessible in [STMO][stmo];
see the [BigQuery cookbook section](../cookbooks/bigquery.md) for more information.
Before analyzing raw ping data,

> **check to make sure there isn't already a derived dataset** made for your purpose.

Reviewer: ```suggestion
you can check if a derived dataset can answer your question.
```
